numpy/typing/tests/data/pass/shape.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, cast, Literal, NewType, TypeVar
+
+import numpy as np
+from typing_extensions import assert_type
+
+Time = NewType("Time", int)
+Series = NewType("Series", int)
```

Comment:
My earlier comment on `NewType` also applies here for the most part. 
Just using `int` directly should be enough.